contents/blog/0-to-hero.md

```diff
@@ -0,0 +1,93 @@
+
+In 2023, we were having a great year in the face of most of [tech struggling](https://www.theatlantic.com/ideas/archive/2023/01/tech-recession-layoffs-google-facebook-microsoft/672798/), and we wanted something to celebrate. We knew there wouldn't be any big external milestones - like a fundraise for example - to break up the year, and we had a _ton_ of stuff we expected our team to build. We therefore wanted to have a bit of a celebration of how things were going. So we increased the offsite budget, went for a fairly fancy hotel in the Caribbean.
+
+As fun as that sounds - a core part of my job is making sure we don't run out of money. Alas.
+
+We turned up and I felt a little *guilty*. The lobby had a _boat_ in it, there was a _private beach_, you could drink _infinite piña coladas_. Add those up, and I knew we were spending $100K more than we _needed_ to.
+
+As any right-minded founder would:
+
+1. I compartmentalized my feelings
+2. I kept refreshing our revenue dashboard
+
+Our growth had been really consistently increasing each month, and it turned out that the month of our offsite was no exception. Within ~2 days we'd increased our annual recurring revenue enough to pay for the _entire_ trip. 
+
+The reason this made me feel so pleased was it demonstrated the efficiency of our business model - we can literally be relaxing on a beach and making money whilst not working. Passive income baby! The reason this happens is that PostHog is a 100% inbound company - we've consistently put useful and cool stuff online (like our products or our content), and because we've done a remarkable-enough job of those things, we now just need to keep building on these to grow through word of mouth. It means we don't need people constantly cold emailing or calling to find more business.
+
+If you fast forward today, 900 companies a week now sign up to PostHog, from hobbyists to Fortune 500s.
+
+<PUT IN GRAPH OF WEEKLY COMPREHENSIVE ORG SIGNUPS>
+
+## Prerequisites
+
+* If you want to go fast, go alone. If you want to go far, go together. You are going to have to handle a literally-endless series of problems. Find a friend! 
+* The ability to ship and get users for new products very fast, from scratch. If you're doing software, you must be very, very good at shipping and marketing software quickly. Frankly, this took me working at a couple of other companies first to get the hang of, it came more quickly for my cofounder.
+* An [idea](https://www.ycombinator.com/library/8g-how-to-get-startup-ideas) of what to work on
+
+## Getting to 10 users - does the product actually function
+
+This was the hardest part.
+
+For every stage of a startup, the earlier you are, the higher the chance of failure. With this in mind, you must be totally geared around failing fast. Hence the prerequisites above.
+
+So, we set off doing this... We wound up [pivoting (ie failing) five times](pivot-to-posthog) before we landed on PostHog. We met the prerequisites!
```

Comment:
My understanding from your pivots was that you were able to build functioning products for the most part, but the issue was getting people to actually pay for them?

Reply:
across various pivots, we failed at:

* people barely using it once
* people not coming back to use it repeatedly
AND
* people not wanting to pay!

I'll clarify this